src/tvheadend/HTSPDemuxer.cpp

```diff
@@ -32,6 +32,8 @@
 #define INVALID_SEEKTIME (-1)
 #define SPEED_NORMAL (1000) // x1 playback speed
 
+#define TVH_STREAM_INDEX_OFFSET (1000)
```

Comment:
What is this?

Reply:
In difference to all others, rds stream is not delivered by tvh, but created by pvr.hts
We need unique stream indexes among all streams - tvh-supplied and pvr.hts created. Easiest way is to add a fixed offset to all streams delivered by tvh and to use numbers less than TVH_STREAM_INDEX_OFFSET for streams created by tvh.

Reply:
Ah, that explains it!